test/hotspot/jtreg/compiler/c2/gvn/TestBitCompressValueTransform.java

```diff
@@ -76,10 +76,7 @@ public long test1(long value) {
 
     @Run(test = "test1")
     public void run1(RunInfo info) {
```

Comment:
While at it, you can also remove all the unused `RunInfo` parameters. But up to you if you want to squeeze that in here as well - it's only a clean-up that is not that important.

Reply:
Sorry for missed this comment......